### PR TITLE
feat: added --version flag for cli

### PR DESCRIPTION
**Pull request by devonwesley**
Changes:
- When requiring `package.json` in the `root/src/index.ts` file it messes with the build path.
- Instead of placing the build files in the `root/build/` folder it creates a `src` folder, changing the build path from `root/build/<PROJECT>` to `root/build/src/<PROJECT>`

Not sure why this is happening or if we want that, but I changed all paths to reflect this new build path.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ import * as util from "util";
 import { keyBy, flatten, chain } from "lodash";
 import { IMethodMapping } from "./router";
 import { resolve } from "path";
+import { version } from "../package.json"; // tslint:disable-line
```
Comment (BelfordZ) [inline src/cli.ts]: I think this only works using the require syntax or dynamic import

Comment (BelfordZ) [reply src/cli.ts]: when you publish to npm, package.json is in `./`, but this file is `./build/src/cli`

Comment (zcstarr) [reply src/cli.ts]: I agree I think dynamic import would be the way to go here perhaps as well 🤔

Comment (devonwesley) [reply src/cli.ts]: Interesting, I switch it back.

Comment (BelfordZ) [reply src/cli.ts]: I'll give you 100 pre-chain-ico BelfordTokens if you also make a test for the cli...

:trollface:

**Comment by BelfordZ** [comment]
image

;)

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     ".node-version"
   ],
   "scripts": {
-    "start": "./build/cli.js",
+    "start": "./build/src/cli.js",
```
Comment (BelfordZ) [inline package.json]:
```suggestion
    "start": "./build/cli.js",
```

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/open-rpc/server-js.git"
   },
-  "main": "./build/index.js",
+  "main": "./build/src/index.js",
```
Comment (BelfordZ) [inline package.json]:
```suggestion
  "main": "./build/index.js",
```

**Pull request merged**